roles/openshift_health_checker/openshift_checks/etcd_volume.py

```diff
@@ -0,0 +1,57 @@
+"""A health check for OpenShift clusters."""
+
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException, get_var
+
+
+class EtcdVolume(OpenShiftCheck):
+    """Ensures etcd storage usage does not exceed a given threshold."""
+
+    name = "etcd_volume"
+    tags = ["etcd", "health"]
+
+    # Default device usage threshold. Value should be in the range [0, 100].
+    default_threshold_percent = 90
+    # Where to find ectd data, higher priority first.
+    supported_mount_paths = ["/var/lib/etcd", "/var/lib", "/var", "/"]
+
+    @classmethod
+    def is_active(cls, task_vars):
+        # TODO: only execute this check on hosts in the 'ectd' group?
+        # Maybe also 'masters' if there are no standalone etcd hosts?
+        return super(EtcdVolume, cls).is_active(task_vars)
+
+    def run(self, tmp, task_vars):
+        mount_info = self._etcd_mount_info(task_vars)
+        available = mount_info["size_available"]
+        total = mount_info["size_total"]
+        used = total - available
+
+        threshold = get_var(
+            task_vars,
+            "r_openshift_health_checker_etcd_device_usage_threshold_percent",
```

Comment:
@sosiouxme thanks for discussing the standard! Let's give it a try.

How about we add:
- In the example inventories, add a section to hold a list of variables used by the checks (they can be all commented-out)
- Somewhere, a conversion from `"openshift_etcd_device_usage_threshold_percent"` to `"r_openshift_health_checker_etcd_device_usage_threshold_percent"` -- @sosiouxme was it part of the discussion where this conversion belongs?
  If we do it at the role level, then the role will need to know about all variables of all existing checks. If we do it here, then it defeats the purpose of having the two names and I'd think why not simply use `"openshift_etcd_device_usage_threshold_percent"` directly?!

BTW, should we consider that "we're doing it wrong" when we read inventory variables like `openshift_deployment_type` and others?!